### PR TITLE
describe principles of languages/countries to CONTRIBUTING guide

### PR DESCRIPTION
**Pull request by asolntsev**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -21,4 +21,17 @@ In order to keep Datafaker slim and focused the project generally attempts to av
...
- 
\ No newline at end of file
+
+## Languages and countries
+
+Localization files are located in folder `src/main/resources`
+1. Country-specific values are stored in files `_COUNTRY.yml` (e.g. `_AL.yml` for Albania)
+   * these are value that are the same in all languages in this country (e.g. phone country code, passport number pattern etc.)
+2. Language-specific values are stored in files `language.yml` (e.g. `it.yml` for Italian)
+   * these are cities, streets, names etc. in that languages.
```
Comment (kingthorin) [inline CONTRIBUTING.md:31]:
```suggestion
   * these are cities, streets, names etc. in that language.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -21,4 +21,17 @@ In order to keep Datafaker slim and focused the project generally attempts to av
 - Minimal size (<1MB)
 - No dependency tree (single level, the library itself) 
 - Fills a need we can't do "simply"
- 
\ No newline at end of file
+
+## Languages and countries
+
+Localization files are located in folder `src/main/resources`
```
Comment (kingthorin) [inline CONTRIBUTING.md:27]:
```suggestion
Localization files are located in folder `src/main/resources`.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -21,4 +21,17 @@ In order to keep Datafaker slim and focused the project generally attempts to av
...
+1. Country-specific values are stored in files `_COUNTRY.yml` (e.g. `_AL.yml` for Albania)
```
Comment (kingthorin) [inline CONTRIBUTING.md:28]:
```suggestion
1. Country-specific values are stored in files `_COUNTRY.yml` (e.g. `_AL.yml` for Albania).
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -21,4 +21,17 @@ In order to keep Datafaker slim and focused the project generally attempts to av
...
+   * these are value that are the same in all languages in this country (e.g. phone country code, passport number pattern etc.)
```
Comment (kingthorin) [inline CONTRIBUTING.md:29]:
```suggestion
   * these are value that are the same in all languages in this country (e.g. phone country code, passport number pattern, etc.).
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -21,4 +21,17 @@ In order to keep Datafaker slim and focused the project generally attempts to av
...
+2. Language-specific values are stored in files `language.yml` (e.g. `it.yml` for Italian)
```
Comment (kingthorin) [inline CONTRIBUTING.md:30]:
```suggestion
2. Language-specific values are stored in files `language.yml` (e.g. `it.yml` for Italian).
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -21,4 +21,17 @@ In order to keep Datafaker slim and focused the project generally attempts to av
...
+   * these are cities, streets, names etc. in that languages.
+3. In files `language-COUNTRY.yml`, we store values specific for this language and country
```
Comment (kingthorin) [inline CONTRIBUTING.md:32]:
```suggestion
3. In files `language-COUNTRY.yml`, we store values specific for this language and country.
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -21,4 +21,17 @@ In order to keep Datafaker slim and focused the project generally attempts to av
...
+   * e.g. `ru-MD.yml` contains Moldovan city names in Russian language (that is widely used in Moldova).
+
+After adding a new country or language, don't forget to update
```
Comment (kingthorin) [inline CONTRIBUTING.md:35]:
```suggestion
After adding a new country or language, don't forget to update:
```

**Review by bodiam — APPROVED** [review]

**Comment by bodiam** [comment]
Nice one, thanks!

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -21,4 +21,17 @@ In order to keep Datafaker slim and focused the project generally attempts to av
...
+* constant `net.datafaker.providers.base.Locality.LOCALES`
+* tests in `LocalityTest.java`
```
Comment (kingthorin) [inline CONTRIBUTING.md:37]:
```suggestion
* constant `net.datafaker.providers.base.Locality.LOCALES`.
* tests in `LocalityTest.java`.
```

**Pull request merged**